Dorian tour story (#2179)
nasa-gibs/worldview: config/default/common/config/metadata/stories/hurricane_dorian_september_2019/step001.md

> This true color corrected reflectance image shows Hurricane Dorian over the Bahamas, taken at approximately 18:35 UTC on 1 September 2019. The image was captured by the Visible Infrared Imaging Radiometer Suite (VIIRS) instrument on the joint NASA/NOAA Suomi-National Polar-orbiting Partnership (Suomi-NPP) satellite.

Reviewer: We capitalize True Color Corrected Reflectance elsewhere in the app, so may want to here/throughout the story.

> Worldview has been consistently providing daily, global satellite imagery from polar orbiting satellites like the joint NASA/NOAA Suomi-NPP, but Worldview is moving into new territory - supplying satellite imagery every 10 minutes from geostationary satellites like GOES East, GOES West and Himawari-8!

Reviewer: ```suggestion
Worldview has been consistently providing daily, global satellite imagery from polar orbiting satellites and now Worldview is moving into new territory - supplying satellite imagery every 10 minutes from geostationary satellites like GOES East, GOES West and Himawari-8!
```

This might be a good step to throw a few links for GOES/Himawari below.

Author: What kind of links?

Reviewer: I was thinking an external link with some mission information, but I'm not finding any good single sources that provide a lot of information.

For Himawari, maybe something like http://www.jma.go.jp/jma/jma-eng/satellite/introduction/satobs.html and/or http://www.data.jma.go.jp/mscweb/en/operation8/index.html

For GOES, maybe https://www.nasa.gov/content/goes-overview/index.html

Author: added some links!
